### ISSUE TITLE
Accessibility Feedback for Design intent - Decision Tools - Sitewide Search UI Consistency

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#23387]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Must...
- Fix mobile breadcrumbs so that they are all consistent with one another. Find a Form and Resources and Support should have the same type of breadcrumb as search e.g. "Home > Find a Form"
- [developers] make the bolded "Search resources and support articles or all of VA.gov" a `<legend>` within a `<fieldset>` that includes the radio options on the resources page

## Should...
- Extend the hyperlink "Coronavirus FAQs" to include "read our" so the purpose for the link is more clear

## Consider
- Including the file size along next to the file type; this helps users with low bandwidth connections or with limited data in deciding if they should download the file. An estimated 25 (VA.gov) - 45% (David Shulkin) of Veterans live in rural areas where a strong data connection may not be present.
- Making page titles consistent; e.g. Find a VA Form and Find Resources and Support OR VA Forms and Resources and Support. The former may be more descriptive.
- Making `<hr>` s consistent. On Find a VA Form and Resources and Support, results are separated with a light grey line `hr`; but on Global Search they don't exist. On Find a VA Form, there is no `hr` above the search field, but one exists on Resources and Support.
- Removing "Powered by Search.gov" and shifting the pagination to the left so it is more easily discoverable with magnification tools. Veterans with low vision who may use external magnification tools not provided within the browser may only see a certain width of the page at any given time. The pagination may be less discoverable than "Powered by Search.gov" (which may give no value to Veterans) if it is flushed right.

## Questions
- Why do we include the form number in front of the link to the form if the link to the form itself includes the form number?
- Would it be possible to reduce the amount of content above the search fields in find a form and resources? It may help the search box to remain above the fold even on older mobile devices like an iPhone SE. e.g.:

<img width="1250" alt="Screen Shot 2021-04-19 at 9 14 57 AM" src="https://user-images.githubusercontent.com/14154792/115242279-b9d05c00-a0ef-11eb-94c1-50a4347ff90f.png">

**Comment by 1Copenut** [comment]
@joshkimux This all looks good. I've just got one point I'd like to update:

> Fix mobile breadcrumbs so that they are all consistent with one another. Find a Form and Resources and Support should have the same type of breadcrumb as search e.g. "Home > Find a Form"

The mobile breadcrumbs are formatted with CSS to be a "back by one" and just show the previous link. If I'm reading the IA correctly, I'd expect each mobile view to have a "< Home" link for breadcrumbs.

Good call to put this one under the Must category.

**Comment by MarciMcGuireGCIO** [comment]
@callen2563 and @mgleo please let me know once you've had time to review and we'll get together to discuss next steps.  Thanks!

**Comment by callen2563** [comment]
@joshkimux Hi Josh, Thank you for reviewing all of this! :) The only part that I don't believe we can fix is the mobile breadcrumb as Home > Resources and Support and some of the longer names don't fit in that space especially on smaller size screens. I pulled that styling from the style guide so I'm wondering if we should leave that for now? @CrystabelReiter do you have any input on this particular issue?

**Comment by CrystabelReiter** [comment]
@callen2563 
Yes so what Trevor says is correct, for mobile we only show the previous link, but what I'm gathering is that we should add Home and the previous link to each mobile screen? 
IMO I think we should leave for now and bring this up to the design system council to review, since this is a styling change and  a documentation update to our breadcrumbs. @joshkimux or @1Copenut Could one of you make a [ticket ](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/new/choose) outlining why we would want to have mobile breadcrumbs have a home link as well?

**Comment by callen2563** [comment]
@CrystabelReiter I did try to put the breadcrumbs in with Home > ... but there wasn't enough page real estate to fit the longer names unless we make them double lined so I'm wondering if maybe we make the back arrow larger on mobile.

**Comment by callen2563** [comment]
@DanielleThierryUSDSVA @bethpotts See @joshkimux comments above ... do you know if we can shorten or move the intro copy on these pages? Particularly on mobile? This would greatly help visibility for the search bar if we could move it up.
